### PR TITLE
iOS 9 Support for Widgets

### PR DESCRIPTION
**Pull request by joewalsh**
- Enable iOS 9 support for widgets
- Update build target to 9.3 per Apple’s recommendation
- Enable Bitcode
- Add activity indicator to Top Read widget to fix inconsistent layout while loading

### CODE REVIEW

File: ContinueReadingWidget/WMFTodayContinueReadingWidgetViewController.swift [hunk 1]
```diff
@@ -21,6 +21,17 @@ class WMFTodayContinueReadingWidgetViewController: UIViewController, NCWidgetPro
     
     override func viewDidLoad() {
         super.viewDidLoad()
+        if #available(iOSApplicationExtension 10.0, *) {
+            
+        } else {
+            titleLabel.textColor = UIColor(white: 1, alpha: 1)
```
Comment (coreyfloyd) [inline ContinueReadingWidget/WMFTodayContinueReadingWidgetViewController.swift]: Why does the styling need to be done differently?

Comment (joewalsh) [inline ContinueReadingWidget/WMFTodayContinueReadingWidgetViewController.swift]: @coreyfloyd view background is black on iOS 9, white on iOS 10

Comment (joewalsh) [inline ContinueReadingWidget/WMFTodayContinueReadingWidgetViewController.swift]: *system provided view background

File: Wikipedia.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -7695,13 +7694,12 @@
...
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.3;
 				ONLY_ACTIVE_ARCH = YES;
 				RUN_CLANG_STATIC_ANALYZER = NO;
 				SDKROOT = iphoneos;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
 				SWIFT_VERSION = 2.3;
-				TARGETED_DEVICE_FAMILY = "1,2";
```
Comment (montehurd) [inline Wikipedia.xcodeproj/project.pbxproj]: @joewalsh 
I think this broke iPad layout... it's now using the phone layout on iPads...

Comment (montehurd) [inline Wikipedia.xcodeproj/project.pbxproj]: Opps hadn't pulled latest... see you've fixed this.

**Pull request merged**